### PR TITLE
Disable elb e2e

### PR DESCRIPTION
**Pull request by erictune**
@kubernetes/goog-testing

### CODE REVIEW

**Comment by erictune** [comment]
Merging this will let us find any other issues with our e2e during the freeze, and to begin merging non-network-related PRs after the freeze ends.

File: hack/jenkins/e2e.sh [hunk 1]
```diff
@@ -134,6 +134,12 @@ GCE_FLAKY_TESTS=(
 GCE_SLOW_TESTS=(
     "SchedulerPredicates\svalidates\sMaxPods\slimit " # 8 min,        file: scheduler_predicates.go, PR:    #13315
     "Nodes\sResize"                                   # 3 min 30 sec, file: resize_nodes.go,         issue: #13323
+    "should\sbe\sable\sto\screate\sa\sfunctioning\sexternal\sload\sbalancer" # Issue with Google internal e2e testing, ref # 24195566
+    "should\sbe\sable\sto\screate\sa\sfunctioning\sexternal\sload\sbalancer\swith\suser-provided\sload\sbalancer\sip" # Issue with Google internal e2e testing, ref # 24195566
+    "should\sbe\sable\sto\schange\sthe\stype\sand\snodeport\ssettings\sof\sa\sservice" # Issue with Google internal e2e testing, ref # 24195566
+    "should\srelease\sthe\sload\sbalancer\swhen\sType\sgoes\sfrom\sLoadBalancer\s->\sNodePort" # Issue with Google internal e2e testing, ref # 24195566
+    "should\scorrectly\sserve\sidentically\snamed\sservices\sin\sdifferent\snamespaces\son\sdifferent\sexternal\sIP\saddresses" # Issue with Google internal e2e testing, ref # 24195566
+    "
```
Comment (ixdy) [inline hack/jenkins/e2e.sh]: extra " is breaking this script:
```
./hack/jenkins/e2e.sh: line 554: syntax error near unexpected token `;'
```

**Comment by brendandburns** [comment]
I would really prefer not to do this.

Let's disable the e2e-gce suite as a requirement for the submit-queue, but leave this.

This is a recipe for never re-activating this.

**Comment by quinton-hoole** [comment]
I disagree @brendandburns.  I think we should merge this PR, and revert it as soon as the quoted buganizer bug is fixed.
If we disable e2e-gce as a requirement for the submit-queue, we'll be losing the signal from 100+ e2e tests, instead of just 5, and probably continue blissfully merging while e2e-gce is broken.

**Comment by brendandburns** [comment]
but we get that signal from e2e-gke

**Comment by quinton-hoole** [comment]
How is it that those tests are passing in e2e-gke?  Different GCE project?  If so, lets run e2e-gce in a new GCE project if that fixes the problem.

**Comment by ArtfulCoder** [comment]
#14455
I suggested that we should have a networking test suite, that can be monitored by the cluster team.
And if we move networking tests out of a test-suite that gates merges, they should move to the networking test suite.

**Comment by erictune** [comment]
Closing since https://github.com/kubernetes/kubernetes/pull/14472 fixed the same problem.

**Pull request closed without merging**